account_followup/models/account_followup.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+# Copyright 2004-2010 Tiny SPRL (<http://tiny.be>)
+# Copyright 2007 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import ValidationError
+
+
+class AccountFollowup(models.Model):
+    _name = 'account_followup.followup'
+    _description = 'Account Follow-up'
+    _rec_name = 'name'
+
+    followup_line = fields.One2many(
+        comodel_name='account_followup.followup.line',
+        inverse_name='followup_id',
+        string='Follow-up',
+        copy=True,
+    )
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string='Company',
+        required=True,
+        default=lambda self: self.env.user.company_id,
+    )
+    name = fields.Char(
+        string="Name",
+        related="company_id.name",
+    )
+
+    @api.multi
+    @api.constrains('company_id')
+    def _check_journal_id(self):
+        if len(self.company_id) > 1:
```

Comment:
This constraint is not correct. The best way to implement this constraint is just simply adding an unique SQL constraint over company_id field, and you can use the same error message.